### ISSUE TITLE
validating/saving model at every epoch

### ISSUE THREAD

**Issue by rohitsaluja22** [TJCVRS/CRNN_Tensorflow#36]
Hey, I want to validate the model at every epoch to avoid overfitting to training data. Presently it just gives training accuracy. Any suggestions how to validate model after each epoch? Or can I save each model instead of last five.

Thanks
Rohit

**Comment by TJCVRS** [comment]
@rohitsaluja22 You can feed the validation data into the model and reuse the weights in the same epoch. The validation mode should feed the phase tensor with the 'test' value.
